.eslintrc

```diff
@@ -1,11 +1,20 @@
 {
-  "extends": "richardpringle",
-  "env": {
-    "es2020": true,
-    "mocha": true
+    "extends": [
+      "richardpringle",
```

Comment:
do we want to keep using this one? feel free to remove it or borrow what makes sense if you like